README.adoc

```diff
@@ -309,7 +309,7 @@ Try enabling detailed logging of the MicroProfile Health feature by adding the [
 
 [role="code_command hotspot" subs="quotes"]
 ----
-#Replace the server configuration file.#
+#Replace the runtime configuration file.#
```

Comment:
```suggestion
#Replace Liberty `server.xml` configuration file.#
```

Reply:
"runtime configuration" may work.

Reply:
Missing "the" so created another suggested commit.

Reply:
I think your suggestion is fine otherwise.